openproblems/tasks/dimensional_reduction/methods/pca.py

```diff
@@ -0,0 +1,17 @@
+from ....tools.decorators import method
+
+import scanpy as sc
+
+@method(
+    method_name="Principle Component Analysis (PCA)",
+    paper_name="On lines and planes of closest fit to systems of points in space",
+    paper_url="https://www.tandfonline.com/doi/abs/10.1080/14786440109462720",
+    paper_year=1901,
+    code_url="https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html",
+    code_version=check_version("scikit-learn"),
+)
+
+
+def pca(adata):
+    
+    sc.tl.pca(adata)
```

Comment:
I don't believe this currently stores the result in `adata.obsm['dimensional_reduction_method']` as specified in your API?

Reply:
Same goes for the other methods

Reply:
The API readme was confusing - I clarified this such that it matches scanpy's convenient, already built-in function outputs.